app/middlewares/user.js

```diff
@@ -0,0 +1,9 @@
+const { validationResult } = require('express-validator');
+
+const { badRequest } = require('../errors');
+
+exports.signUpMiddleware = (req, _, next) => {
+  const { errors } = validationResult(req);
+  if (errors.length > 0) next(badRequest(errors.map(err => err.msg)));
+  return next();
+};
```

Comment:
You could make the middleware something more general so that it can be used with any scheme.
For example: 
`const checkValidationResult = { ... }  -> signUpMiddleware logic`
`exports.validateSchema = schema => [checkSchema(schema), checkValidationResult]`